File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuri/NUR0101U01GrdMasterInfo.java
package nta.med.data.model.ihis.nuri;

public class NUR0101U01GrdMasterInfo {
	private String codeType;
    private String codeTypeName;
    private String adminGubun;
	public NUR0101U01GrdMasterInfo(String codeType, String codeTypeName,
			String adminGubun) {
		super();
		this.codeType = codeType;
		this.codeTypeName = codeTypeName;
		this.adminGubun = adminGubun;
	}
	public String getCodeType() {
		return codeType;
	}
	public void setCodeType(String codeType) {
		this.codeType = codeType;
	}
	public String getCodeTypeName() {
		return codeTypeName;
	}
	public void setCodeTypeName(String codeTypeName) {
		this.codeTypeName = codeTypeName;
	}
	public String getAdminGubun() {
		return adminGubun;
	}
	public void setAdminGubun(String adminGubun) {
		this.adminGubun = adminGubun;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuri/NUR0101U01GrdMasterInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuri
{
    [Serializable]
	public class NUR0101U01GrdMasterInfo
	{
		private String _codeType;
		private String _codeTypeName;
		private String _adminGubun;
		private String _dataRowState;

		public String CodeType
		{
			get { return this._codeType; }
			set { this._codeType = value; }
		}

		public String CodeTypeName
		{
			get { return this._codeTypeName; }
			set { this._codeTypeName = value; }
		}

		public String AdminGubun
		{
			get { return this._adminGubun; }
			set { this._adminGubun = value; }
		}

		public String DataRowState
		{
			get { return this._dataRowState; }
			set { this._dataRowState = value; }
		}

		public NUR0101U01GrdMasterInfo() { }

		public NUR0101U01GrdMasterInfo(String codeType, String codeTypeName, String adminGubun, String dataRowState)
		{
			this._codeType = codeType;
			this._codeTypeName = codeTypeName;
			this._adminGubun = adminGubun;
			this._dataRowState = dataRowState;
		}

	}
}